content/pages/education/apply-for-education-benefits/application/1990.md

```diff
@@ -4,40 +4,11 @@ entryname: 1990-edu-benefits
 layout: page-react.html
 description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
 ---
-<div id="main">
-  <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-  id="va-breadcrumbs">
-    <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-      <li><a href="/">Home</a></li>
-      <li><a href="/education/">Education</a></li>
-      <li><a aria-current="page" href="/education/apply-for-education-benefits/">Apply for Education Benefits</a></li>
-    </ul>
-  </nav>
-  <div class="section">
-    <div id="react-root">
-      <div class="loading-message">
-        <div class="loading-indicator-container">
-          <div class="loading-indicator" role="progressbar" aria-valuetext="Please wait while we load the application for you." tabIndex="0"></div> Please wait while we load the application for you.
-        </div>
-      </div>
-    </div>
-  </div>
-  <!-- Edu Benefits Application End -->
-
-  <!-- Maintenance Page Start -->
-
-  <!-- <div class="main home" role="main">
-    <div class="section main-menu">
-      <div class="row">
-        <div class="small-12 columns">
-          <div class="csp-inline-patch-application">
-          <h3>We’re sorry. The education benefits application is currently down while we fix a few things. We’ll be back up as soon as we can.</h3>
-          <h4>In the meantime, you can call 1-877-222-VETS (<a href="tel:+18772228387">1-877-222-8387</a>), Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (<abbr title="eastern time">ET</abbr>) and press 2 to complete this application over the phone.</h4>
-          <a href="/"><button>Go back to Vets.gov</button></a>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>-->
-  <!-- Maintenance Page End -->
-</div>
```

Comment:
I removed all of this commented code because we have actual template files now for rendering maintenance messaging.

Reply:
_Insert Cyberman joke here._